### PR TITLE
Update README content to make it more relevant

### PR DESCRIPTION
**Pull request by WasabiFan**
The current wiki content was last updated 2 years ago! This should get the readme into a state that gets people where they need to go while not overwhelming them. The goal is for the readme to be its own _TL;DR_.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+
+### Searching for code? Have a look at all of the other repositories in the [ev3dev GitHub organization](https://github.com/ev3dev).
+
+This repo just hosts the "meta content" that spans our many codebases and subprojecs. The code itself is managed granularly on a per-project level, and each gets its own repo.
+
+**Central code repositories:**
+- [ev3dev kernel](https://github.com/ev3dev/ev3-kernel): The foundational Linux-based kernel that runs ev3dev.
+- [LEGO device drivers](https://github.com/ev3dev/lego-linux-drivers): Drivers that provide control over LEGO MINDSTORMS devices and related hardware.
+- [Brickman](https://github.com/ev3dev/brickman): Graphical brick manager for ev3dev.
+- [ev3dev website](https://github.com/ev3dev/ev3dev.github.io): The ev3dev website and documentation.
```
Comment (ddemidov) [inline README.md]: How about the ev3dev-lang repo? Some people only come here to be able to program their ev3 in a real programming language.

Comment (WasabiFan) [inline README.md]: Ah, good point -- I think I'll add a section for that.

Comment (WasabiFan) [inline README.md]: @ddemidov Just added that section and corresponding repo link -- can you take a look?

File: README.md [hunk 2]
```diff
@@ -16,6 +20,7 @@ This repo just hosts the "meta content" that spans our many codebases and subpro
 - [LEGO device drivers](https://github.com/ev3dev/lego-linux-drivers): Drivers that provide control over LEGO MINDSTORMS devices and related hardware.
 - [Brickman](https://github.com/ev3dev/brickman): Graphical brick manager for ev3dev.
 - [ev3dev website](https://github.com/ev3dev/ev3dev.github.io): The ev3dev website and documentation.
+- [ev3dev-lang language bindings](https://github.com/ev3dev/ev3dev-lang): A centralized collection of libraries for controlling ev3dev functionality from user space code.
```
Comment (ddemidov) [inline README.md]: This does not feel right, can we simplify the wording? Something like:
```
A centralized collection of libraries for programming your ev3dev robots.
```

Comment (WasabiFan) [inline README.md]: Yes, that's much better. I must be tired :wink: Updated.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,29 @@
 The ev3dev meta repo
 ====================
-This repository doesn't actually have any code. It hosts [downloads](https://github.com/ev3dev/ev3dev/releases),
-the ev3dev [wiki](https://github.com/ev3dev/ev3dev/wiki) and the [issues](https://github.com/ev3dev/ev3dev/issues) tracker.
 
-To learn more about ev3dev, check out our home page at <http://www.ev3dev.org>.
+This repository doesn't actually have any code. It hosts [downloads](https://github.com/ev3dev/ev3dev/releases) and the project-wide  [issues](https://github.com/ev3dev/ev3dev/issues) tracker.
```
Comment (dlech) [inline README.md]: How about "SD card image downloads" instead of just "downloads"?

You could also stick "tracker" in the link with "issues"

File: README.md [hunk 4]
```diff
@@ -1,9 +1,29 @@
...
 
+Our website hosts docs, guides and tutorials to help you get up-and-running with ev3dev as quickly as possible. Check out the [getting started guide](http://www.ev3dev.org/docs/getting-started/) if you're looking for a place to jump in.
+
+### Want to get started writing your own programs? Look through the [list of available libraries](http://www.ev3dev.org/docs/libraries/).
+
+We have many contributors that maintain libraries to make it as easy as possible to interface with ev3dev from your code. There's a [centralized repo](https://github.com/ev3dev/ev3dev-lang) with a "unified" library set as well as an overall [list of available libraries](http://www.ev3dev.org/docs/libraries/). Just pick a language, find the corresponding library, and dive into that library's examples. Or, you can always access the drivers directly if your language isn't already implemented as a pre-made library; there's  information on this style on our [docs landing page](http://www.ev3dev.org/docs/).
+
+### Searching for code? Have a look at all of the other repositories in the [ev3dev GitHub organization](https://github.com/ev3dev).
```
Comment (dlech) [inline README.md]: We should probably make it clear that this "code" is for working on the ev3dev OS itself as opposed to example code for programming your robot.

File: README.md (hunk 1 above)
Comment (dlech) [inline README.md]: brickstrap is another core repository.

Also, if you want to build a kernel then ev3dev-build-scripts is what you want. In fact, this is probably the repository that I hand out links for the most, so it might even merit it's own section. "Want to compile a kernel? Go here..."

File: README.md [hunk 5]
```diff
@@ -1,9 +1,29 @@
...
+This repo just hosts the "meta content" that spans our many codebases and subprojecs. The code itself is managed granularly on a per-project level, and each gets its own repo.
+
+**Central code repositories:**
+- [ev3dev kernel](https://github.com/ev3dev/ev3-kernel): The foundational Linux-based kernel that runs ev3dev.
+- [LEGO device drivers](https://github.com/ev3dev/lego-linux-drivers): Drivers that provide control over LEGO MINDSTORMS devices and related hardware.
+- [Brickman](https://github.com/ev3dev/brickman): Graphical brick manager for ev3dev.
+- [ev3dev website](https://github.com/ev3dev/ev3dev.github.io): The ev3dev website and documentation.
+- [ev3dev-lang language bindings](https://github.com/ev3dev/ev3dev-lang): A centralized collection of libraries for programming your ev3dev robots.
+
+### Need support or found a bug? Read through our [support page](http://www.ev3dev.org/support/) to find the best place to ask.
```
Comment (dlech) [inline README.md]: This section should be closer to the top. This is probably what most people are looking for.

**Comment by WasabiFan** [comment]
@dlech Changes made.

**Comment by dlech** [comment]
Thanks!
